samples/nrf9160/mqtt_simple/src/main.c

```diff
@@ -449,6 +449,16 @@ static void button_handler(uint32_t button_states, uint32_t has_changed)
  */
 static int modem_configure(void)
 {
+
+	/* Turn off LTE power saving features for a more responsive demo. Also,
+	 * request power saving features before network registration. Some
+	 * networks rejects timer updates after the device has registered to the
+	 * LTE network.
+	 */
```

Comment:
If `CONFIG_LTE_AUTO_INIT_AND_CONNECT` is enabled, still will still be an issue, but not much we can do about that